TestCases/compliance-level-3/0093-feel-at-literals/0093-feel-at-literals.dmn

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<definitions namespace="http://www.montera.com.au/spec/DMN/0093-feel-at-literals"
+             name="0093-feel-at-literals"
+             id="_i9fboPUUEeesLuP4RHs4vA"
+             xmlns="https://www.omg.org/spec/DMN/20191111/MODEL/"
+             xmlns:di="http://www.omg.org/spec/DMN/20180521/DI/"
+             xmlns:dmndi="https://www.omg.org/spec/DMN/20191111/DMNDI/"
+             xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <description>FEEL @ literals</description>
+
+    <decision name="test_001" id="_test_001">
+        <variable name="test_001"/>
+        <literalExpression>
+            <text>@"foo" = null</text>
```

Comment:
My understanding is that you want to test that an At-literal must conform to being one of the temporal cases from the specification.

I'm pretty sure in analogous cases, we tested for returning null in the test case, with an helper flag.

I believe is meaningful to keep the same strategy to test errors.

Hence, I believe this test shall be amended in the FEEL expression to be, more simply:
```
<text>@"foo"</text>
```

and test case for expecting
```
<value xsi:nil="true"/>
```

Reply:
Thanks @tarilabs . Fair call.  Amended.

Reply:
this was actioned with 50e8432, marking https://github.com/dmn-tck/tck/pull/391#discussion_r584096252 as resolved.